config/formats.ts

```diff
@@ -1805,50 +1805,77 @@ export const Formats: import('../sim/dex-formats').FormatList = [
 			'Vulpix-Alola', 'Yanma', 'Yanmega',
 		],
 		validateSet(set, teamHas) {
-			let pokemoves = 0;
-			const problems: string[] = [];
-			const moves = [];
+			const problems = [];
+			const pokemovesArray = [];
+			// First, identify all Pokemoves
 			if (set.moves?.length) {
-				if (set.moves.length > this.ruleTable.maxMoveCount) {
-					problems.push(`${set.name} has ${set.moves.length} moves, which is more than the limit of ${this.ruleTable.maxMoveCount}.`);
-					return problems;
-				}
-				for (const [i, moveid] of set.moves.entries()) {
+				for (const moveid of set.moves) {
 					const pokemove = this.dex.species.get(moveid);
-					if (!pokemove.exists) continue;
-					if (pokemove.isNonstandard &&
-						!(this.ruleTable.has(`+pokemontag:${this.toID(pokemove.isNonstandard)}`) ||
-							this.ruleTable.has(`+pokemon:${pokemove.id}`) ||
-							this.ruleTable.has(`+basepokemon:${this.toID(pokemove.baseSpecies)}`))) {
-						problems.push(`${pokemove.isNonstandard} Pok\u00e9mon are not allowed to be used as Pokemoves.`);
-					}
-					if (this.ruleTable.isRestrictedSpecies(pokemove) || this.ruleTable.isBannedSpecies(pokemove)) {
-						problems.push(`${pokemove.name} is unable to be used as a Pokemove.`);
+					if (pokemove.exists) {
+						// This is a Pokemove
+						pokemovesArray.push(moveid);
```

Comment:
```suggestion
					if (pokemove.exists) {
						// This is a Pokemove
						pokemovesArray.push(moveid);
```

This is objectively wrong, moves that exist at this step are _not_ pokemoves.

Reply:
Yeah makes sense, very sorry for wasting your time. I'll try to fix it myself.